.github/workflows/updatecli.yaml

```diff
@@ -27,6 +27,14 @@ jobs:
         with:
           go-version: '1.19.6'
```

Comment:
Unrelated, but updating the go-version to stable here would prevent the need to keep this up to date.